spec/asyncapi.md

```diff
@@ -1599,10 +1618,22 @@ components:
           type: string
   servers:
     development:
-      url: development.gigantic-server.com
+      url: {stage}.gigantic-server.com:{port}
```

Comment:
`mdv` markdown linter [is failing](https://github.com/asyncapi/spec/runs/5832670366?check_suite_focus=true) since you are not adding proper quotation to this. In YAML, curly brackets are use for maps, so the quotes are needed.

This will fix CI
```suggestion
      url: "{stage}.gigantic-server.com:{port}"
```